common/web/keyboard-processor/src/text/outputTarget.ts

```diff
@@ -23,6 +23,7 @@ export class TextTransform implements Transform {
   readonly deleteLeft: number;
   readonly deleteRight: number;
   readonly erasedSelection: boolean;
+  id: number;
```

Comment:
Brought to you by `noImplicitAny`.  TS began complaining about its type not being available, as it _is_ referenced in some of the predictive-text integration code.

On the base type `Transform`, `id` is marked as optional: 

https://github.com/keymanapp/keyman/blob/963587f7968fe517e71aec99a7483434e99845d1/common/models/types/index.d.ts#L226-L234

Within this type (`TextTransform`), it was not declared at all:

https://github.com/keymanapp/keyman/blob/963587f7968fe517e71aec99a7483434e99845d1/common/web/keyboard-processor/src/text/outputTarget.ts#L21-L26

As a result, TS inferred that `TextTransform` simply didn't support `.id`.  I'm not 100% sure why `noImplicitAny` was the trigger, because it previously did infer type, but... 🤷